lib/io/adc.cpp

```diff
@@ -10,8 +10,8 @@ Adc::Adc(const uint8_t pin, hyped::core::ILogger &logger) : pin_(pin), logger_(l
   char buf[100];
   snprintf(buf, sizeof(buf), "/sys/bus/iio/devices/iio:device0/in_voltage%i_raw", pin_);
   file_ = open(buf, O_RDONLY);
-  if (file_ < 0) {
-    logger_.log(hyped::core::LogLevel::kFatal, "Unable to open ADC file");
+  if (file_ < 0) { /* TODO: log that there is a problem reading raw voltage for pin_*/
```

Comment:
?

Reply:
shit. Formatted and got conflict. Will fix now